tests/test_policies.py

```diff
@@ -72,6 +80,31 @@ def test_policy(env_name, policy_name, extra_overrides):
         + extra_overrides,
     )
 
+    # Additional config override logic.
```

Comment:
FYI: This (and in general the idea of testing policies x datasets) scales as n². Not nice. I think we need to think of an O(1) solution like validating that datasets have a certain data key format, and policies can handle this format.